src/client.js

```diff
@@ -73,8 +78,11 @@ function Client(sockets, name, config) {
 		});
 	}
 }
+util.inherits(Client, EventEmitter);
 
+/** An emitter that emits both locally and through network */
 Client.prototype.emit = function(event, data) {
+	EventEmitter.prototype.emit.apply(this, arguments);
```

Comment:
If `Client` inherits from `EventEmitter`, shouldn't [`this.emit(this, arguments);`](https://nodejs.org/api/events.html#events_emitter_emit_event_arg1_arg2) be enough?

Where did you find this line? I feel like such way of doing so would be necessary if `EventEmitter` would be something completely external to `Client`, that is, if there was no inheritance mechanism between those two.

If I'm right, then only one is needed, the ugly `prototype.emit.apply` call vs. the overkill-looking inheritance mechanism. If I'm wrong, slap me in the face.